projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -306,30 +432,62 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
         }
     }
 
+    /**
+     * Scrolls by one item into the appropriate next direction.
+     * ```typescript
+     * this.parentVirtDir.scrollNext();
+     * ```
+     * @param index
+     */
     public scrollNext() {
         this.scrollTo(this.state.startIndex + 1);
     }
 
+    /**
+     * Scrolls by one item into the appropriate previous direction.
```

Comment:
@alexalex972 "Scrolls by one item into the appropriate previous direction. For `"horizontal"` orientation that will be the the left column and for `"vertical"` that is the upper row."